content/en/blog/_posts/2022-05-20-non-graceful-node-shutdown.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Introducing Non-Graceful Node Shutdown Alpha"
+date: 2022-05-20
+slug: kubernetes-1-24-non-graceful-node-shutdown-alpha
+---
+
+**Authors** Xing Yang and Yassine Tijani (VMware)
+
+Kubernetes v1.24 introduces alpha support for [Non-Graceful Node Shutdown](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2268-non-graceful-shutdown). This feature allows stateful workloads to failover to a different node after the original node is shutdown or in a non-recoverable state such as the hardware failure or broken OS.
+
+## How is this different from Graceful Node Shutdown
+
+You might have heard about the [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown) capability of Kubernetes,
+and are wondering how the Non-Graceful Node Shutdown feature is different from that. Graceful Node Shutdown
+allows Kubernetes to detect when a node is shutting down cleanly, and handle that situation appropriately.
+A Node Shutdown can be "graceful" only if the node shutdown action can be detected by the kubelet ahead
+of the actual shutdown. However, there are cases where a node shutdown action may not be detected by
+the kubelet. This could happen either because the shutdown command does not trigger the systemd inhibitor
+locks mechanism that kubelet relies upon, or because of a configuration error
+(the `ShutdownGracePeriod` and `ShutdownGracePeriodCriticalPods` are not configured properly).
+
+Graceful node shutdown relies on Linux-specific support. The kubelet does not watch for upcoming
+shutdowns on Windows nodes (this may change in a future Kubernetes release).
+
+When a node is shutdown but without the kubelet detecting it, Pods on that node
```

Comment:
I think we can change `Pod` -> `pod` in the doc.

Reply:
Done